doc/guide/go_quickstart.hpp

```diff
@@ -0,0 +1,241 @@
+/**
+ * @file go_quickstart.hpp
+ * @author Yashwant Singh Parihar
+
+@page go_quickstart mlpack in Go quickstart guide
+
+@section go_quickstart_intro Introduction
+
+This page describes how you can quickly get started using mlpack from Go and
+gives a few examples of usage, and pointers to deeper documentation.
+
+This quickstart guide is also available for
+@ref cli_quickstart "the command-line", @ref python_quickstart "Python"
+and @ref julia_quickstart "Julia".
+
+@section go_quickstart_install Installing mlpack
+
+Installing the mlpack bindings for Go is somewhat ticker; you can download
```

Comment:
```suggestion
Installing the mlpack bindings for Go is somewhat time-consuming as the library must be built; you can run the following code:
```
Maybe this is clearer?

Reply:
I am really sorry, I was making blunder here.
Thank you this suggestion. :)